Shared/Supporting Files/Views/SampleDetailView.swift

```diff
@@ -109,3 +112,10 @@ struct SampleDetailView: View {
 extension SampleDetailView: Identifiable {
     var id: String { sample.nameInUpperCamelCase }
 }
+
+private extension URL {
```

Comment:
Instead of extending `URL`, consider extending `Sample` with a property for the GitHub URL of the sample. That seems like a more appropriate place for the URL.